packages/lib-content/test/setup.js

```diff
@@ -0,0 +1,46 @@
+import chai from 'chai'
+import dirtyChai from 'dirty-chai'
+import sinonChai from 'sinon-chai'
+import { JSDOM } from 'jsdom'
+
+chai.use(dirtyChai)
+chai.use(sinonChai)
+global.expect = chai.expect
+
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>', { url: 'https://localhost'})
+const { window } = jsdom
+
+
+function copyProps(src, target) {
+  const props = Object.getOwnPropertyNames(src)
+    .filter(prop => typeof target[prop] === 'undefined')
+    .map(prop => Object.getOwnPropertyDescriptor(src, prop))
+  Object.defineProperties(target, props)
+}
+
+class ResizeObserver {
+  disconnect() {}
+  observe() {}
+  unobserve() {}
+}
+
+class IntersectionObserver {
+  disconnect() {}
+  observe() {}
+  unobserve() {}
+}
+
+
+
+window.IntersectionObserver = IntersectionObserver
+window.ResizeObserver = ResizeObserver
+window.scrollTo = () => true
+
+global.dom = jsdom
+global.window = window
+global.self = global.window
```

Comment:
This statement is not included in other FEM packages, but I had to add it here 🤔 Several components throw an error `self is not defined` in the testing env without it, and I could not for the life of me figure out the exact cause. I suspect it has to do with `next/link` in a library outside of a Next app, but I'll continue to investigate while building out lib-content.